sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyOptions.java

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.cryptography;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents optional parameters for cryptographic operations.
```

Comment:
It's not always the case that `Options` implies 'optional' - indeed, in this case there are some required parameters. It would be better to change this text so that people don't build that impression of Options types. Saying something instead, such as "A class containing various configuration parameters that can be applied when performing cryptographic operations."